resources/lang/en/model_validation.php

```diff
@@ -24,18 +24,35 @@
     'too_long' => ':attribute exceeded maximum length - can only be up to :limit characters.',
     'wrong_confirmation' => 'Confirmation does not match.',
 
-    'beatmap_discussion' => [
+    'beatmap_discussion_post' => [
+        'discussion_locked' => 'Discussion is locked.',
+        'first_post' => 'Can not delete starting post.',
+    ],
+
+    'beatmapset_discussion' => [
+        'beatmap_missing' => 'Timestamp is specified but beatmap is missing.',
+        'beatmapset_no_hype' => "Beatmap can't be hyped.",
+        'hype_requires_null_beatmap' => 'Hype must be done on general all section.',
+        'locked' => 'Discussion is locked.',
+        'mapper_note_wrong_user' => 'Only beatmap owner can post mapper note.',
+
+        'beatmap_id' => [
+            'missing' => 'Associated beatmap is somehow gone.',
+            'mismatch' => "Difficulty and beatmap don't match.",
```

Comment:
shouldn't this be more like `beatmapset mismatch between discussion and beatmap` or something?